sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -207,6 +208,18 @@ $OPT --sycl-xocc -S -O3 -xoccIRDowngrader \
   "${TMP_PATH}_${SOURCE_FILE_NAME}_kernels-linked.simple.xpirbc" -o \
   "${TMP_PATH}_${SOURCE_FILE_NAME}_kernels-linked.opt.ll"
 
+# Our bash file containing kernel property environment variables is placed in
+# the systems temporary directory (or wherever the Clang driver is told is the
+# temporary directory) and is postfixed with the source files name. It's
+# generated by the kernelPropertiesGen OPT pass.
+source $KERNELPROP
+
+if [[ ! -z "${SYCL_XOCC_MANUAL_EDIT}" ]]; then
+  echo "Please edit ${TMP_PATH}_${SOURCE_FILE_NAME}_kernels-linked.opt.ll"
+  echo "Kernel name: ${KERNEL_NAME_ARRAY[0]}"
+  read -p "Press return to resume compilation flow"
```

Comment:
You could automate this with `emacsclient` or `gvim --remote` https://superuser.com/questions/132634/is-there-an-equivalent-to-emacsclient-for-vim/232860 :-)
I guess there might be the same stuff for Visual Code or Eclipse or...